client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -38,6 +39,14 @@ export const SiteItemThumbnail = ( { site, ...props }: SiteItemThumbnailProps )
 		} );
 	}
 
+	if ( site.is_coming_soon ) {
+		const style = {
+			width: props.width || DEFAULT_THUMBNAIL_SIZE.width,
+			height: props.height || DEFAULT_THUMBNAIL_SIZE.height,
+		};
+		return <SiteComingSoon siteName={ site.name } style={ style } lang={ site.lang } />;
```

Comment:
I think an alternative to exporting `DEFAULT_THUMBNAIL_SIZE` would be to render `<SiteComingSoon>` _inside_ the `<SiteThumbnail>`, like the existing fallbacks. That way the `<SiteComingSoon>` could be styled to simply fill its parent.

Maybe that wouldn't work though because you need access to the actual dimensions as part of the SVG markup 🤔

Reply:
I considered this initially but thought that `<SiteThumbnail/>` already had a lot going on and that it would confusing to accept quite a lot of props for `mShots` and then just return something totally different. We'd still need dimensions and also have to pass in a flag, e.g. `isComingSoon`, too, which didn't feel right.